tools.json

```diff
@@ -1,4 +1,4 @@
 {
   "tools": {
   }
-}
```

Comment:
remove tools.json changes

Reply:
Done

Reply:
remove this file from your changes, exclude it in this PR. Dont delete the content of the file

Reply:
check now